CMakeLists.txt

```
@@ -116,8 +123,14 @@ if(SLEIGHHOME_DEFAULT)
 	endif()
 endif()
 
+if(NOT DEFINED CMAKE_INSTALL_LIBDIR)
+	set(CMAKE_INSTALL_LIBDIR lib64 CACHE PATH "Output directory for libraries.")
+endif()
+
 install(TARGETS core_ghidra DESTINATION "${RIZIN_INSTALL_PLUGDIR}")
 if(BUILD_SLEIGH_PLUGIN)
 install(TARGETS asm_ghidra DESTINATION "${RIZIN_INSTALL_PLUGDIR}")
 install(TARGETS analysis_ghidra DESTINATION "${RIZIN_INSTALL_PLUGDIR}")
 endif()
+install(FILES ${PUBLIC_HEADERS} DESTINATION ${CMAKE_INSTALL_PREFIX}/include)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/rz_ghidra.pc DESTINATION ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/pkgconfig)
```

Don't explicitly specify `CMAKE_INSTALL_PREFIX` and use ""s, same in the line above.
```suggestion
install(FILES "${CMAKE_CURRENT_BINARY_DIR}/rz_ghidra.pc" DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
```

---

Done, thanks!